src/utils/conversion.ts

```diff
@@ -1030,6 +1031,23 @@ export function permissionGroupToAgentGroup(
   );
 }
 
+/**
+ * @hidden
+ */
+export function agentGroupToPermissionGroup(agentGroup: AgentGroup): PermissionGroup {
```

Comment:
This function should be renamed considering the type will be renamed on the other PR